### PR TITLE
Self-hosting not officially supported

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Related to issue https://github.com/OfficeDev/office-js/issues/502.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ The NPM package for Office.js is intended as a way for you to obtain your own (n
 
 2. If you need offline access to the Office.js APIs (for example, to facilitate offline debugging).
 
+**Note**: Officially, Outlook doesn't support this. However, if you choose to proceed, you may also need to self-host a copy of Microsoft Ajax or other dependencies as needed.
```
Comment (ElizabethSamuel-MSFT) [inline README.md]:
```suggestion
**Note**: Officially, Outlook doesn't support this. However, if you choose to proceed, you may also have to self-host a copy of Microsoft Ajax or other dependencies as needed.
```

**Review by exextoc — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ The NPM package for Office.js is intended as a way for you to obtain your own (n
 
 2. If you need offline access to the Office.js APIs (for example, to facilitate offline debugging).
 
+**Note**: Officially, Outlook doesn't support this. However, if you choose to proceed, you may also have to self-host a copy of Microsoft Ajax or other dependencies as needed.
```
Comment (Zlatkovsky) [inline README.md]: Why just "Outlook"?  None of the Office Add-ins support this...

**Review by Zlatkovsky — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sumurthy) [reply README.md]: I think "support" could mean different thing to Outlook. It is true that Microsoft Store will not accept any add-in that doesn't reference CDN directly for Office-js library. My understanding is that Outlook is saying the offline files won't work technically (which is hard for me to understand). I'll check with product team and get back.

**Review by sumurthy — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -32,6 +32,16 @@ Best practices for using the Office.js NPM package include:
...
 
+### Support
+
+The Office.js CDN is the official, supported source for Office Add-ins. This means that for the NPM package the following are not guaranteed.
+
+- Frequency of updates
+
+- Timely support for issues and bugs
+
+Also, Outlook does not support offline scenarios due to a network access requirement for dependencies like the Microsoft Ajax library.
```
Comment (sumurthy) [inline README.md]: Should we say "offline hosting of JavaScript library" instead of "offline scenario"

Comment (ElizabethSamuel-MSFT) [inline README.md]:
```suggestion
Also, Outlook does not support hosting Office.js offline due to a network access requirement for dependencies like the Microsoft Ajax library.
```

File: README.md [hunk 4]
```diff
@@ -130,7 +140,7 @@ For example, to use a `1.1.2-beta-next.0` version, use the following references:
     @microsoft/office-js@1.1.2-beta-next.0/dist/office.d.ts
 
 
-![Using the NPM package with Script Lab](https://github.com/OfficeDev/office-js/blob/release/.github/images/script-lab-substitute-refs.png)
+![Using the NPM package with Script Lab](https://raw.githubusercontent.com/OfficeDev/office-js/release/.github/images/script-lab-substitute-refs.png)
```
Comment (ElizabethSamuel-MSFT) [inline README.md]:
```suggestion
![Using the NPM package with Script Lab](https://github.com/OfficeDev/office-js/blob/release/.github/images/script-lab-substitute-refs.png)
```

**Review by wandyezj — APPROVED** [review]

**Pull request merged**